src/test/java/com/google/api/codegen/testdata/php/php_library.baseline

```diff
@@ -437,18 +443,18 @@ class LibraryServiceGapicClient
      *     @type string $serviceAddress
      *           The address of the API remote host. May optionally include the port, formatted
      *           as "<uri>:<port>". Default 'library-example.googleapis.com:443'.
-     *     @type string|array $auth
+     *     @type string|array $credentials
```

Comment:
Could we port the documentation update from [here](https://github.com/googleapis/gax-php/blob/master/src/ApiCore/GapicClientTrait.php#L157) over as well?

Reply:
`FetchAuthTokenInterface` and `CredentialsWrapper` need imports. While checking to see if those existed, I noticed we now have a few other imports which are no longer used and can probably go.

```php
use Google\ApiCore\Call;
use Google\Auth\CredentialsLoader;
use Grpc\Channel;
use Grpc\ChannelCredentials;
```

Reply:
Updated imports